src/bundle/Resources/public/scss/_badges.scss

```diff
@@ -1,68 +1,76 @@
 .ibexa-badge {
-    padding: calculateRem(4px);
+    height: calculateRem(24px);
+    padding: calculateRem(4px) calculateRem(10px);
     font-size: calculateRem(16px);
-    text-decoration: none;
+    font-size: $ibexa-text-font-size-small;
+    line-height: calculateRem(16px);
+    font-weight: normal;
+    border: calculateRem(1px) solid $ibexa-color-dark;
+    border-radius: calculateRem(12px);
```

Comment:
new 12px value?

Reply:
It's the same case as in dropdown which partly shares design :)
https://github.com/ibexa/admin-ui/pull/383#discussion_r838592103